{"uuid": "da9a3300-7b10-4c25-b49a-ebdeb0259f70", "children": ["51473fa8-3689-470c-971b-917df17f70b4"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x562a8840effa <unknown>\n#1 0x562a87ecd970 <unknown>\n#2 0x562a87f1f385 <unknown>\n#3 0x562a87f1f5b1 <unknown>\n#4 0x562a87f6e3c4 <unknown>\n#5 0x562a87f452bd <unknown>\n#6 0x562a87f6b70c <unknown>\n#7 0x562a87f45063 <unknown>\n#8 0x562a87f11328 <unknown>\n#9 0x562a87f12491 <unknown>\n#10 0x562a883d642b <unknown>\n#11 0x562a883da2ec <unknown>\n#12 0x562a883bda22 <unknown>\n#13 0x562a883dae64 <unknown>\n#14 0x562a883a1bef <unknown>\n#15 0x562a883fd558 <unknown>\n#16 0x562a883fd736 <unknown>\n#17 0x562a8840de76 <unknown>\n#18 0x7f7ec769caa4 <unknown>\n#19 0x7f7ec7729c3c <unknown>\n\n", "trace": "  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QATEST/QATEST/src/tests/ui_tests/conftest.py\", line 30, in driver\n    WebDriverWait(driver, 10).until(\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1743024799910, "stop": 1743024811773}], "start": 1743024799910, "stop": 1743024811779}